### PR TITLE
fix: u-boot on Raspberry Pi for 32 bit mode

### PR DESCRIPTION
**Pull request by TheYoctoJester**

### CODE REVIEW

File: meta-mender-raspberrypi/recipes-bsp/u-boot/u-boot-raspberrypi.inc [hunk 1]
```diff
@@ -6,9 +6,12 @@ BOOTENV_SIZE:rpi ?= "0x4000"
 # use of boot.scr and cmdline.txt.
 MENDER_UBOOT_AUTO_CONFIGURE:rpi = "0"
 
+SRC_URI:append:rpi = " file://memory_size.cfg"
```
Comment (lluiscampos) [inline meta-mender-raspberrypi/recipes-bsp/u-boot/u-boot-raspberrypi.inc:9]: Isn't this configuration uboot specific?
```suggestion
SRC_URI:append:rpi:mender-uboot = " file://memory_size.cfg"
```

Comment (TheYoctoJester) [reply meta-mender-raspberrypi/recipes-bsp/u-boot/u-boot-raspberrypi.inc:9]: @lluiscampos it is `u-boot` specific, hence in the `u-boot` recipe, but its not Mender specific. So not tied to the `mender-uboot` feature.

Technically both of those *could* go to `meta-raspberrypi`, but I want to unblock our users first, then care about upstream.

File: meta-mender-raspberrypi/recipes-bsp/u-boot/patches/0001-pci-fdt-ignore-checks-for-address-range.patch
Comment (ghost) [inline meta-mender-raspberrypi/recipes-bsp/u-boot/patches/0001-pci-fdt-ignore-checks-for-address-range.patch:1]: > Commit 78126711cb2167931775cff84acfcba17b07689f in u-boot upstream
> adds "pci: Check region ranges are addressable". This blocks the
> booting process on Raspberry Pi platforms running in 32bit mode.

I can't find the commit that this paragraph is talking about. Which repo is it?

**Review by ghost — APPROVED** [review]
I can't really say much about the validity of the address patch, but seems sensible. 🙂

**Pull request merged**